### PR TITLE
added test for comment update endpoint

### LINKED ISSUE #134: Creating Test for update comment end point
**Is your feature request related to a problem? Please describe.**
When update comment endpoint is created, tests will be needed.

**Describe the solution you'd like**
Create and incorporate tests for the update comment endpoint

**Additional context**
Waiting on #132 . Communicate with the dev for that issue first.

### PR DESCRIPTION
**Pull request by ElliotW1999**
Fixes #134 

## Proposed Changes

  - Created a test for updating comments

### CODE REVIEW

**Comment by ljcnz** [comment]
Could probably do with two more test cases; just for when you send an empty body in the update request (should return 400) and for trying to update a comment that doesn't exist (should return 404) @gracecocks thoughts?

**Comment by ElliotW1999** [comment]
Added both tests, please re-review

File: server/test/integration/post.spec.js [hunk 1]
```diff
@@ -495,6 +495,125 @@ describe('Posts API', () => {
...
+    const commentUrl = '/api/comments/';
+
+    // update the comment
+    const updatedComment = {
+      commentBody: 'Second Body',
+      parentID: createdPost._id,
+    };
+
+    const response3 = await supertest(app)
+      .put(commentUrl)
```
Comment (jacoballen4534) [inline server/test/integration/post.spec.js:568]: This is not actually calling any of our endpoints. This is calling /api/comments (with no id) instead of an invalid id.
You would expect the body to come back with an object
`{ message: 'Cast to ObjectId failed for value "<non_existant_id" at path "_id" for model "Comment" }`, however, this is just going to call a non-existent endpoint, (coincidentally giving the same error code)
Instead, commentUrl  should be '/api/comments/non_existant_id'

Comment (ElliotW1999) [reply server/test/integration/post.spec.js]: I've incorporated both changes - this test now gives the message you've mentioned, but validating this explicitly is a bit convoluted (prettier doesn't like it).  
Is it necessary to add this validation, or is the response.status equal to 404 sufficient?

Comment (jacoballen4534) [reply server/test/integration/post.spec.js]: Checking status code should be alright, that is also consistent with the rest of the tests.
